td-agent/Rakefile

```diff
@@ -244,6 +244,15 @@ class BuildTask
     @download_task = download_task
   end
 
+  def make_wix_version_numer(version)
+    if version.split("~")[1]
```

Comment:
It may be better to restrict the number of splitting explicitly.

I mean `version.split("~", 2)`